cmd/agent/gui/checks.go

```diff
@@ -270,22 +270,24 @@ func listChecks(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	// Get wheels
 	pyIntegrations, err := getPythonChecks()
 	if err != nil {
 		log.Errorf("Unable to compile list of installed integrations: %v", err)
 		w.Write([]byte("Unable to compile list of installed integrations."))
 		return
 	}
 
+	// Get python wheels
 	integrations = append(integrations, pyIntegrations...)
 
 	// Get go-checks
 	goIntegrations := core.GetRegisteredFactoryKeys()
 	integrations = append(integrations, goIntegrations...)
 
 	// Get jmx-checks
-	integrations = append(integrations, check.JMXChecks...)
+	for integration, _ := range check.JMXChecks {
```

Comment:
why change the append ?

Reply:
because I changed the implementation of `JMXChecks` to be a set now (implemented with a `map[string]struct{}`), so I don't think the old append works anymore....